SimpleServlet/src/main/java/com/skankhunt220/api/controller/UserController.java

```diff
@@ -9,68 +9,64 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-import org.bson.Document;
-
 import main.java.com.skankhunt220.entities.User;
 import main.java.com.skankhunt220.service.UserService;
 
-@WebServlet("/Example") // servlet url
-public class RequestHandler extends HttpServlet {
+@WebServlet("/User") // servlet url
+public class UserController extends HttpServlet {
 	private static final long serialVersionUID = 1L;
 	private UserService userService;
-	private PrintWriter printWriter;
+	private User user;
 
 	private void sendResponse(String message, HttpServletRequest request, HttpServletResponse response)
 			throws IOException {
 
 		response.setContentType("text/html");
-		printWriter = response.getWriter();
-		User user = new User();
+		try (PrintWriter writer = response.getWriter()) {
+			user = new User();
```

Comment:
Why does the `sendResponse` method work with filling user fields? As i said This method is only responsible for sending a response. Fill user fields in the method, where the request was comed